CK3toEU4/Data_Files/blankMod/output/common/cultures/!.expanded_culture_groups.txt

```diff
@@ -19,6 +19,16 @@ scandinavian = {
 		dynasty_names = { Arnórsson Ásgeirsson Benediktsson Bjarnasen Björnsson Eggers Eldjárn Hafstein Hansen
 		Hermannsson Johánesson Jónsson Magnússon Stefánsson Steinthosson Thórdarson Thórhallson Thorláksson Thurs }
 	}
+	elsinorian = {
```

Comment:
Just curious where you got this word as I cannot find it with a quick google search. It's certainly less cumbersome than anglo-scandinavian.